docs/parts/controllers.md

```diff
@@ -223,6 +223,52 @@ quit
 
 Now that your controller is trusted, it should automatically connect with your Raspberry Pi when they are both turned on.  If your controller fails to connect, run the bluetoothctl steps again to reconnect.
 
+## Advanced Joystick Mode
+
+The Advanced Joystick mode uses button combinations to allow more control on the car.
+In order to use the Advanced Joystick mode, you must update the manage.py file and  replace the line get_controller_js() with get_controller_js(Vobject=V)
```

Comment:
Please wrap code parts in backtics, such as: `some code`

Reply:
Hi, thanks for your feedback, i believe i have resolved the point now with my new controller.py  ( 21/10/2019). 
Regards,